src/features/views/components/ViewColumnDialog/ColumnGallery/utils.ts

```diff
@@ -0,0 +1,14 @@
+import choices, { CHOICES } from '../choices';
+
+export const filterChoicesByMode = (
+  isRestrictedMode: boolean,
+  category: { choices: CHOICES[] }
```

Comment:
I wasn't clear enough in my previous comment. I mean this line should only read: `choices: CHOICES[]` (with some subsequent changes made to the body of this function, and to where the function is called from).

There is no need to pass in the entire category objects.

Reply:
ah okay! I hope I fixed it right this time!